src/App.css

```diff
@@ -16,15 +16,28 @@ body {
 }
 
 .View--Login {
+  height: 100vh;
   background-image: linear-gradient(to bottom, #fff 400px, transparent 100%),
-    url(https://portal.kidsfirstdrc.org/static/media/background-science.68317e4e.jpg);
+    url(./assets/background-science.jpg);
+    .center.column{
+      min-width: 300px !important;
+    }
 }
 
-.Card--Login {
-  @extend .Card;
-  width: 620px;
+.ui.button.button--login{
+    margin-bottom: 10px;
+    .auth0-logo{
+      width: 70px;
+    }
+
+  }
+  .ui.buttons .or.button--login{
```

Comment:
It looks like this is unused?

Reply:
the LoginCard (probably need to more accurately name the component) uses it